lms/static/sass/shared/_fields.scss

```diff
@@ -12,7 +12,7 @@
         border-radius: 3px;
 
         span {
-            color: $gray-l1;
+            color: $gray-d1;
```

Comment:
A quick platform search for `mode-placeholder` (which contains this `span`) only appear on pages that related the profile or profile fields, so I'm assuming this "shared" style is equally scoped. I'd also be okay moving it into the "non-shared" Sass partial, if you'd prefer a new style there.

Also, `$gray` is also passing here, so I'll use it instead.